### PR TITLE
Reduce Windows machine size

### PR DESCRIPTION
**Pull request by janisz**
## High-level description

We can use smaller machines for our CI, to save some money.

## Corresponding DC/OS tickets (required)

  - [D2IQ-66575](https://jira.d2iq.com/browse/D2IQ-66575)
  - [D2IQ-66580](https://jira.d2iq.com/browse/D2IQ-66580)  

## Related tickets (optional)

  - [D2IQ-66576](https://jira.mesosphere.com/browse/D2IQ-66576)

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [ ] Change log from the last version integrated (this should be a link to commits for easy verification and review): [example](https://github.com/dcos/dcos-mesos-modules/compare/f6fa27d7c40f4207ba3bb2274e2cfe79b62a395a...6660b90fbbf69a15ef46d0184e36755881d6a5ae)
  - [ ] Included a test which will fail if code is reverted but test is not. If there is no test please explain.
  - [ ] Test Results: [link to CI job test results for component]
  - [ ] Code Coverage (if available): [link to code coverage report]

### CODE REVIEW

File: test_util/main.tf [hunk 1]
```diff
@@ -87,7 +106,10 @@ module "dcos" {
   num_public_agents  = "1"
 
   dcos_instance_os        = "centos_7.5"
-  bootstrap_instance_type = "m4.xlarge"
```
Comment (janisz) [inline test_util/main.tf:90]: The default is `t2.medium` so let's stick with it.

**Comment by janisz** [comment]
@mesosphere-mergebot request-review @jongiddy @jeschkies @DominikDary

**Comment by janisz** [comment]
@mesosphere-mergebot changelog-not-required OSS internal

File: test_util/main.tf [hunk 2]
```diff
@@ -76,6 +93,8 @@ module "dcos" {
   tags {
     owner = "${var.owner}"
     expiration = "${var.expiration}"
+    build_id = "${var.build_id}"
```
Comment (DominikDary) [inline test_util/main.tf:96]: @orsenthil does this works as expected and can you find the corresponding cost details in cost explorer?

**Comment by janisz** [comment]
Since we can't get dcos build I tests in manually and it's working

```
-- Docs: https://docs.pytest.org/en/latest/warnings.html
======= 53 passed, 1 skipped, 7 xfailed, 127 warnings in 1109.61 seconds =======
```

**Comment by janisz** [comment]
@mesosphere-mergebot request-review @orsenthil

File: test_util/main.tf (hunk 2 above)
Comment (orsenthil) [reply test_util/main.tf:96]: @DominikDary  - I don't how it correlates with cost details. But I will find out.

Comment (orsenthil) [reply test_util/main.tf:96]: And this conversation on measurement only indirectly related to the motivation here, which is about reducing the size of the instance. So, +1 with this change if the reduced instance size meets our testing criteria.

File: test_util/main.tf [hunk 3]
```diff
@@ -50,15 +50,32 @@ variable "dcos_license_key_contents" {
...
 # Used to determine your public IP for forwarding rules
 data "http" "whatismyip" {
   url = "http://whatismyip.akamai.com/"
 }
 
 resource "random_string" "password" {
-  length = 6
-  special = true
-  override_special = "-"
+  length = 12
```
Comment (orsenthil) [inline test_util/main.tf:77]: @janisz  - I think, we will run into problems here.

* Terraform / aws has limits on size of the string the characters allowed.

Could you please test this locally before using it?

Comment (orsenthil) [reply test_util/main.tf:77]: By testing locally, I meant, try to test it 3-5 times.

Comment (janisz) [reply test_util/main.tf:77]: Good point, I'll change to 6 and cluster ID so names will be shorter.

Comment (janisz) [reply test_util/main.tf]: Updated can you take another look

Comment (janisz) [reply test_util/main.tf]: Ouch, it was already on train. I'll fix it later but all runs local + 3 on CI passed so I think we do not need to change it.

**Pull request merged**